README.md

```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
 =====================================================
 
-Ghub is a library that provides basic support for using the Github
+Ghub is a library that provides basic support for using the GitHub
 REST (v3) and GraphQL (v4) APIs from Emacs packages.  It abstracts
 access to API resources using only a handful of functions that are
 not resource-specific.
 
-It also handles the creation, storage and use of access tokens using a
-setup wizard, to make it easier for users to get started and to reduce
-the support burden imposed on package maintainers.  It also comes with
+It also handles the creation, storage, and use of access tokens by using a
+setup wizard to make it easier for users to get started and to reduce
+the support burden imposed on package maintainers and comes with
 a comprehensive manual to address the cases when things don't just
 work as expected.
 
 Ghub is intentionally limited to only provide these two essential
-features — basic request functions and guided setup — to avoid being
+features—basic request functions and guided setup—to avoid being
```

Comment:
I believe that both styles are used and maybe even that the one that you are proposing is the one people use you know what they are doing. Again it looks really weird to me and after the Oxford comma experience I don't feel like doing a similar experiment again and just stick to what I like.

When using a proportional font the "connected" style actually looks okay, but this text also appears in other places where a monospace font is used and then the connected style looks bad.